### ISSUE TITLE
graph from readme

### ISSUE THREAD

**Issue by jviquerat** [hmomin/TD3-Bipedal-Walker#1]
Hi,

Thank you for sharing this work.
Could you detail what exactly is plotted on the dynamic histogram in the main readme ?

Best

**Comment by hmomin** [comment]
Sure thing, Jonathan.

In the [`main.py`](https://github.com/hmomin/TD3-Bipedal-Walker/blob/main/src/main.py) code, I ran 64 total trials, each one for 2,000 episodes.

The way I calculated evaluation returns is I would train the agent on each step through a single episode and then run an entire evaluation episode without any training steps using deterministic actions from the policy -> I would log the total reward of that evaluation episode. This is also made clear in the [`main.py`](https://github.com/hmomin/TD3-Bipedal-Walker/blob/main/src/main.py) file.

I now had a 64 x 2,000 matrix of data with dimensions (number of trials, number of episodes per trial). Taking the mean and standard deviation along the first dimension gave me two different 1 x 2,000 vectors that I used to create the [episode vs. reward graph on the README](https://camo.githubusercontent.com/c4cd603c9d66a9154204ba0a657e6a427fcb7b4552e89f7a0f285549be60ad02/68747470733a2f2f6472336e676c3739377a3534762e636c6f756466726f6e742e6e65742f6269706564616c2d77616c6b65722d36342d747269616c2d6176672e706e67).

In [the dynamic histogram](https://user-images.githubusercontent.com/72412509/154755506-ae973863-6cb5-455c-b16d-6ad58e9ae39a.mp4), what you're seeing at each time-step is a 64 x 1 slice in the second dimension of that matrix of data. So, for a single episode (Episode X in the title of the graph), what does a histogram of all 64 trials look like for their evaluation performance just on that specific episode? That's what's being plotted.

Hope this helps,
Momin
